### DISCUSSION TITLE
Merged a PR with Co-author, but didn't get the Pair Extraordinaire achievement

### DISCUSSION THREAD

**Discussion by SKY-ALIN** [community/community#67384] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello! I made a PR [SKY-ALIN/Alinsky.tech#3](https://github.com/SKY-ALIN/Alinsky.tech/pull/3) that had a Co-author, but I didn't get the Pair Extraordinaire achievement. Please let me know if I made any mistakes there.

**Comment by seyLu** [comment, marked as answer]
#### Hi @SKY-ALIN,

To ensure you earn the "Pair Extraordinaire" achievement, make sure to add a co-author below the commit message before merging to the `main` branch.

```bash
# Commit message

Co-authored-by: SKY-ALIN <SKY-ALIN@users.noreply.github.com>
Co-authored-by: Someone-else <Someone-else@users.noreply.github.com>
```

You can read more on [Github Blog: Commit together with co-authors](https://github.blog/2018-01-29-commit-together-with-co-authors/).
